pkg/install/squash.go

```diff
@@ -14,13 +14,9 @@ import (
 )
 
 var (
-	SquashRepoName  = "quay.io/solo-io"
-	SquashName      = "squash"
-	SquashImageName = "squash"
-
+	SquashName    = "squash"
```

Comment:
I'd like to merge this portion of the change if you don't mind rebasing

Reply:
Sure. I'll update this PR.